share/transition_scripts/openldap_to_389ds/openldap_to_389ds.rb

```diff
@@ -192,7 +192,7 @@ def convert_user(attrs)
   unless attr[:objectclass].member?('organizationalUnit')
     if dn.end_with?(oldgroupdn)
       new_dn = dn.gsub(oldgroupdn, newgroupdn)
-      ldifs389[new_dn] = convert_group(attr, newgroupdn)
+      ldifs389[new_dn] = convert_group(attr, olduserdn)
```

Comment:
Why was this not picked up by the acceptance tests?  Should the test be updated?